metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/ExecutionAggregator.java

```diff
@@ -12,27 +11,29 @@
 
 /**
  * An interface defining the methods needed to aggregate RunExecution's
+ * @param <T> The type of execution, example: RunExecution or ValidationExecution
  * @param <M> The aggregated metric from Metrics
  * @param <E> The execution metric from RunExecution
  */
-public interface RunExecutionAggregator<M, E> {
+public interface ExecutionAggregator<T, M, E> {
 
     M getMetricFromMetrics(Metrics metrics);
-    E getMetricFromRunExecution(RunExecution runExecution);
+    E getMetricFromExecution(T execution);
```

Comment:
This class and methods seem fundamental, a little bit of documentation would help me to better understand them.